locale/en.json

```diff
@@ -523,6 +523,8 @@
   "BASIC_INPUTS": "BASIC FIELDS",
   "COMPOUND_INPUTS": "COMPOUND FIELDS",
   "SPECIAL_INPUTS": "SPECIAL FIELDS",
+  "SIGHTING_SEARCH_RESULT_PRIMARY_TEXT": "{date} sighting in {region}",
```

Comment:
that might sound better in some cases but it doesn't make sense grammatically in all cases. because this is a specified date it can take many forms, such as `2004`, `May 2004`, `May 2, 2004`, or `May 2, 2004 at 8:15`. So we would have "sighting in mozambique on 2004" for example...